todo-app/src/stores/TodoStore.js

```diff
@@ -0,0 +1,108 @@
+import {observable, action, useStrict, runInAction} from 'mobx'
+import Moment from 'moment'
+
+
+useStrict(true)
+
+export default class TodoStore{
+
+    constructor(){
+      this.loadTodos();   // without constructor it won't work, you need to use componentDidMount
```

Comment:
What does this comment mean? Why are you not calling `loadTodos` in `componentDidMount`?

Reply:
Actually it was for myself to future check, 
I meant that if you won't use constructor you have to use "ComponentDidMount" utility.
In class you did it in both ways, and I didn't want to miss any of them :)
I know I will need to remember all the stuff

Reply:
Ah okay, got it. As Joost pointed out in class, it's probably better to use `componentDidMount`, so that the lifecycle of the data is directly related to the lifecycle of the UI.